Rajitha/recursion-practice.js

```diff
@@ -0,0 +1,60 @@
+/****************************************************
+* Create a function to get the factoral of a number.
+* Input: factoral(5)  Output: 120
+****************************************************/
+let fact = 1;
+const factorial = num => {
+  if (num > 1) {
+    fact = fact*num;
+    num--;
+    fact = factorial(num);
+  }
+  return fact;
+};
+
+console.log(factorial(7));
+
+/*********************************************************
+* Create a function to get the power of a number.
+* Multiply the first input x(second input) number of times
+* Input: power(2, 5)
+* Output: 32
+* reason: 2x2x2x2x2
+**********************************************************/
+let check = 1;
+let result = 1;
+const numPower = (num,exp) => {
+  if (check <= exp){
+    result = result * num;
+    check++;
+    result = numPower(num,exp);
+  }
+  return result;
+}
+console.log(numPower(2,5));
+
+/***************************************************
+* Reverse a string
+* Input: reverse("Recursion Sucks")
+* Output: "skcuS noisruceR"
+****************************************************/
+let revString = '';
+const  reverseString = str => {
```

Comment:
please create different test/edge cases. Don't rely only on the example given. - RY